src/txHandler/TxHandler.interface.ts

```diff
@@ -39,5 +39,7 @@ export interface ISimpleTxHandler extends INotifierManager, ApprovalHandlerInter
 }
 
 export interface IBatchTxHandler extends INotifierManager {
-  handleBatchTransaction: (operations: Operation[], options?: TransactionOptions) => Promise<any>;
+  addOperation: (operation: TxOperation) => void;
+  removeOperation: (index: number) => void;
```

Comment:
to simplify a lot, one could only allow removing the last operation for now

Reply:
Yes agree, this is stg that we'll refine during e2e tests